packages/babel-plugin-transform-typescript/src/index.js

```diff
@@ -325,13 +330,16 @@ export default declare(
           // class transform would transform the class, causing more specific
           // visitors to not run.
           path.get("body.body").forEach(child => {
-            if (child.isClassMethod()) {
+            if (child.isClassMethod() || child.isClassPrivateMethod()) {
```

Comment:
Note: this doesn't actually fix anything, but I'm ok with keeping it since we could add new things to the `method` handler, which could be also allowed with private methods.

Reply:
Yes, I left it for consistency and for possible further changes